applications/leetcoin.md

```diff
@@ -107,81 +185,81 @@ Please also provide the GitHub accounts of all team members. If they contain no
-| 11. | Logging                | Server logging with morgan and winston                                                                                                                                                                | 
+|-------:|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                | Apache 2.0                                                                                                                                                                                            |
+|    0b. | Documentation          | We will provide the documentation for the repository code as well as a tutorial for publishing/designing a user-generated coding problem and test suites.                                             |
+|    0c. | Testing Guide          | Best practices for developing test suites to run code against and a FAQ.                                                                                                                              |
+|    0d. | Article                | Announcement blog post explaining how LeetCoin can be used for upgrading web3 developer skills, improve security and help with properly vetting dev core-team candidates to minimize technical risks. |
+|     1. | Design Document        | Description of architecture and interface                                                                                                                                                             |  
+|     2. | User Authentication    | Create a wallet based authentication system for problem builders as well as users with Web3 Onboard                                                                                                   |  
+|     3. | Online Judge           | Online judge that compiles Ink! contract code and runs it against test suites comparing the actual output with the expected ones based on the different test cases                                    |  
+|     4. | Frontend problems list | A simple table that displays all available problems on the platform                                                                                                                                   | 
+|     5. | Frontend code editor   | Code editor for the problems                                                                                                                                                                          |  
+|     6. | First Ink! Problem Set | Develop first 5 Ink! problems working with core teams of popular DeFi protocols                                                                                                                       | 
```

Comment:
After talking with Ink! ecosystem developer, we realized that defining 10 problems is a little difficult and want to consider 5 quality ones first